src/components/Bar.jsx

```diff
@@ -9,6 +9,8 @@ import Drawer from 'components/Drawer'
 import Nav from 'components/Nav'
 import Claudy from 'components/Claudy'
 import SupportModal from 'components/SupportModal'
+import { flowRight as compose } from 'lodash'
```

Comment:
Where is it from? I don't see any lodash dependecies :s

Reply:
You're right, you do not have `lodash` in your dependencies but it seems your dependencies are using it, this is why it works.

``` cat yarn.lock | grep lodash
    lodash "^4.14.0"
    lodash "^4.17.4"
    lodash "^4.17.4"
    lodash "^4.17.4"
    lodash "^4.17.4"
    lodash "^4.17.4"
    lodash "^4.17.4"
    lodash "^4.2.0"
    lodash "^4.17.4"
    lodash "^4.2.0"
    lodash "^4.17.4"
    lodash "^4.2.0"
```

A bit weird that it possible to do something like that. Do you prefer `translate()(connectToStore(Bar))` ? It is possible :)